openreview/conference/builder.py

```diff
@@ -740,9 +740,13 @@ def recruit_reviewers(self, emails = [], title = None, message = None, reviewers
 
         if remind:
             remind_reviewers = list(set(reviewers_invited_group.members) - set(reviewers_declined_group.members) - set(reviewers_accepted_group.members))
-            for reviewer in remind_reviewers:
-                name =  re.sub('[0-9]+', '', reviewer.replace('~', '').replace('_', ' ')) if reviewer.startswith('~') else 'invitee'
-                tools.recruit_reviewer(self.client, reviewer, name,
+            for reviewer_id in remind_reviewers:
```

Comment:
is there a way to reproduce this issue with a test?

It is difficult to see what you are fixing.

Reply:
@melisabok I have added a test and a description of the change to the PR.